Add Zine client and description to the feature support matrix

kirk-marple opened the pull request:
<!-- Provide a brief summary of your changes -->

## Motivation and Context
<!-- Why is this change needed? What problem does it solve? -->

## How Has This Been Tested?
<!-- Have you tested this in a real application? Which scenarios were tested? -->

## Breaking Changes
<!-- Will users need to update their code or configurations? -->

## Types of changes
<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation update

## Checklist
<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [X] I have read the [MCP Documentation](https://modelcontextprotocol.io)
- [ ] My code follows the repository's style guidelines
- [ ] New and existing tests pass locally
- [ ] I have added appropriate error handling
- [X] I have added or updated documentation as needed

## Additional context
<!-- Add any other context, implementation notes, or design decisions -->

dend commented on docs/clients.mdx:
```diff
@@ -179,7 +181,7 @@ This page provides an overview of applications that support the Model Context Pr
 [Zed]: https://zed.dev
 [Zencoder]: https://zencoder.ai
-[HyperAgent]: https://github.com/hyperbrowserai/HyperAgent
+[Zine]: https://www.zine.ai
```
Is the removal of HyperAgent intentional @kirk-marple?

kirk-marple commented:
Just fixed the sorting of it. Moved it up to the right spot.

On Sun, Sep 7, 2025 at 10:21 PM Den Delimarsky ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In docs/clients.mdx
> <https://github.com/modelcontextprotocol/modelcontextprotocol/pull/1419#discussion_r2329161482>
> :
>
> > @@ -179,7 +181,7 @@ This page provides an overview of applications that support the Model Context Pr
>  [Witsy]: https://github.com/nbonamy/witsy
>  [Zed]: https://zed.dev
>  [Zencoder]: https://zencoder.ai
> -[HyperAgent]: https://github.com/hyperbrowserai/HyperAgent
> +[Zine]: https://www.zine.ai
>
> Is the removal of HyperAgent intentional @kirk-marple
> <https://github.com/kirk-marple>?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/modelcontextprotocol/modelcontextprotocol/pull/1419#pullrequestreview-3194995891>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADHW7NTVQ2ZLSXIPRXPW5V33RUG5XAVCNFSM6AAAAACFSZQ6MGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTCOJUHE4TKOBZGE>
> .
> You are receiving this because you were mentioned.Message ID:
> <modelcontextprotocol/modelcontextprotocol/pull/1419/review/3194995891@
> github.com>
>

kirk-marple commented:
> @kirk-marple - to clarify, the PR diff is still showing that a client is being removed. Just checking if this is intentional? Assuming just ordering was the issue here?

Not intentional to remove anything.

But I think it's not showing you the whole diff, and the corresponding '+' above.

I'd just moved their line into its properly sorted spot above.

<img width="1089" height="671" alt="image" src="https://github.com/user-attachments/assets/4bc46951-3130-4d8d-a1ab-a8235a628d01" />
